frontend/components/product-list/sections/FilterableProductsSection/FacetFilter.tsx

```diff
@@ -1,61 +1,63 @@
-import { getProductListPath } from '@helpers/product-list-helpers';
+import {
+   getProductListPath,
+   getRefinementDisplayType,
+} from '@helpers/product-list-helpers';
 import { ProductList, ProductListType } from '@models/product-list';
 import * as React from 'react';
 import { RefinementList } from './RefinementList';
 import { RefinementMenu } from './RefinementMenu';
+import { useInstantSearch } from 'react-instantsearch-hooks-web';
+import { uiStateToQueryString } from '@components/common/InstantSearchProvider';
+import { RefinementDisplayType } from '@models/product-list/types';
+import { useProductListItemType } from './useProductListItemType';
 
 type FacetFilterProps = {
    attribute: string;
    productList: ProductList;
 };
 
 export function FacetFilter({ attribute, productList }: FacetFilterProps) {
+   const { indexUiState } = useInstantSearch();
+   const queryString = React.useMemo(
+      () => uiStateToQueryString(indexUiState, ['facet_tags.Item Type']),
+      [indexUiState]
+   );
+   const itemType = useProductListItemType(productList);
    const createItemTypeURL = React.useCallback(
       (itemType: string) => {
-         return getProductListPath({
-            ...productList,
-            type: ProductListType.DeviceItemTypeParts,
-            itemType,
-         });
+         const path = getProductListPath(
+            {
+               ...productList,
+               type: ProductListType.DeviceParts,
```

Comment:
We don't need to redefine the `type` field anymore, it's not changing anything. You can just pass productlist